2014-12-06  Magnus Granberg  <zorry@gentoo.org>

	Return the output_buffer

--- a/backend/zobcs/pym/actions.py	2014-10-20 03:51:42.000000000 +0200
+++ b/backend/zobcs/pym/actions.py	2014-12-06 12:40:26.319364154 +0100
@@ -1692,11 +1692,8 @@ def action_info(settings, trees, myopts,
 			unset_vars.append(k)
 	if unset_vars:
 		append("Unset:  "+", ".join(unset_vars))
-	append("")
-	append("")
-	writemsg_stdout("\n".join(output_buffer),
-		noiselevel=-1)
-	del output_buffer[:]
+
+	return False, output_buffer
 
 	# If some packages were found...
 	if mypkgs:
